pkg/api/http/http.go

```diff
@@ -696,15 +696,15 @@ func (a *api) getStateStoreWithRequestValidation(reqCtx *fasthttp.RequestCtx) (s
 	storeName := a.getStateStoreName(reqCtx)
 
 	if a.universal.CompStore().StateStoresLen() == 0 {
-		err := apierrors.NotConfigured(storeName, string(contribMetadata.StateStoreType)+" store", map[string]string{"appID": a.universal.AppID()}, codes.FailedPrecondition, nethttp.StatusInternalServerError, "ERR_STATE_STORE_NOT_CONFIGURED", kiterrors.CodePrefixStateStore+kiterrors.CodeNotConfigured)
+		err := apierrors.NotConfigured(storeName, string(contribMetadata.StateStoreType)+" store", nil, codes.FailedPrecondition, nethttp.StatusInternalServerError, "ERR_STATE_STORE_NOT_CONFIGURED", kiterrors.CodePrefixStateStore+kiterrors.CodeNotConfigured)
```

Comment:
Why is the metadata removed here?

Reply:
It didn't exist previously until my PR, but when I added it in it was failing a bunch of integration tests, so I removed it to minimize the test failures. Its back now. Still working on the test failures tho.